docs/monitoring/tag-aliases-support.md

```diff
@@ -0,0 +1,107 @@
+title: Tag Aliases
+description: Tag aliases are name-value pairs used to identify a group of tag values i.e. alias for a group of tags. For example, user can create a tag alias `env:prod` which identifies a set of hosts. 
+
+## What are Tag Aliases?
+
+Tag aliases are name-value pairs used to identify a group of tag values i.e. alias for a group of tags. For example, user can create a tag alias `env:prod` which identifies a set of hosts running in prod setup. So user can define this tag alias and map it set of `os.host` tag values. Tag aliases can be defined from UI, configured in agent or automatically sent by agents.
+
+## Automatic Tags Aliases from agent
+
+The Infra Agent automatically collects following tag aliases and sends them periodically to Sematext Cloud. These tag aliases are purged periodically. It is not recommended to use these names in configured/user-defined tag aliases.
```

Comment:
"these names" - which names?

Reply:
Fixed. Moved the text to the general section:
```
All tag aliases except user-defined aliases are sent periodically by agent and are also purged periodically in Sematext Cloud.
```